packages/sitecore-jss-nextjs/src/services/component-props-service.ts

```diff
@@ -105,21 +105,21 @@ export class ComponentPropsService {
    * Go through layout service data, check all renderings using displayName, which should make some side effects.
    * Write result in requests variable
    * @param {Object} params params
-   * @param {PlaceholdersData} params.placeholders
+   * @param {PlaceholdersData} [params.placeholders]
    * @param {FetchFunctionFactory<NextContext>} params.fetchFunctionFactory
    * @param {LayoutServiceData} params.layoutData
    * @param {NextContext} params.context
    * @param {ComponentPropsRequest<NextContext>[]} params.requests
    * @returns {ComponentPropsRequest<NextContext>[]} array of requests
    */
   collectRequests<NextContext>(params: {
```

Comment:
Not a new issue, but I noticed this function is exposed publicly. Should we mark this as private to avoid having to worry about signature changes affecting our public API surface going forward? Looks like these functions could all be private: `collectRequests`, `execRequests`, `flatRenderings`

Reply:
Good idea, I did it.